examples/stable_diffusion_xl/gm/modules/diffusionmodules/denoiser.py

```diff
@@ -1,15 +1,13 @@
 # reference to https://github.com/Stability-AI/generative-models
-
 from gm.util import append_dims, instantiate_from_config
 
 import mindspore as ms
 from mindspore import Tensor, nn, ops
 
 
 class Denoiser(nn.Cell):
-    def __init__(self, weighting_config, scaling_config):
+    def __init__(self, scaling_config):
```

Comment:
maybe not to change the exposed interface (espeicailly for deletion) unless we have a good test suite for them

Reply:
Reverted